functions/src/get-reading/util.ts

```diff
@@ -1,19 +1,12 @@
 import {PurpleAirReading} from './types';
 
-/**
- * Get the readings subcollection path from a sensor's doc ID
- * @param docId - document ID of the sensor in the sensors collection
- * @returns the path to the readings subcollection for a sensor
- *
- */
-const readingsSubcollection: (docId: string) => string = (docId: string) =>
-  `/sensors/${docId}/readings`;
-
 /**
  * Converts PurpleAir's confidence value into percent difference
  * @param confidence - confidence value from PurpleAir, between 0 and 100
- * @returns meanPercentDifference value, or NaN if value lost in calculation.
- * Any input that results in NaN means that the meanPercentDifference is high
+ * @returns meanPercentDifference value, or 2 (the maximum possible percent difference ) if the value is lost in calculation.
```

Comment:
What makes 2 the maximum possible percent difference?

Reply:
Percent difference is the absolutely value of the difference over the average, like so for values `a` and `b`: `(2*|a-b|)/(a+b)`. When `a` and `b` are the same, the percent difference is `0` since `a-b` is 0. Then consider when `a` and `b` are SUPER different, let's say `a` is tiny and `b` is huge. In the limit as `b` goes to infinity and `a` is epsilon, `|a-b|/(a+b)` goes to 1, so the whole fraction goes to 2.

A fun [Desmos ](https://www.desmos.com/calculator/gxbiwg23ef)to see this as well